addons/advanced_ballistics/XEH_postInit.sqf

```diff
@@ -22,3 +22,17 @@ if (!GVAR(extensionAvailable)) exitWith {
 };
 */
 [] call FUNC(initializeTerrainExtension);
+
+if (!hasInterface) exitWith {};
+
+["SettingsInitialized", {
+    //If not enabled, dont't add PFEH
+    if (!GVAR(enabled)) exitWith {};
+
+    // Register fire event handler
+    ["firedPlayer", DFUNC(handleFired)] call EFUNC(common,addEventHandler);
+    ["firedPlayerNonLocal", DFUNC(handleFired)] call EFUNC(common,addEventHandler);
```

Comment:
Nah, that's not possible. There'sa additional conditions inside the if of line 28, and there's no easy way to know in advance if they'll be true or false, except if GVAR(simulateForSnipers) == false and GVAR(simulateForGroupMembers) = false. I think it's a bit to hard for a too small gain